lib/chromic_pdf.ex

```diff
@@ -34,49 +34,59 @@ defmodule ChromicPDF do
 
-  For some apparent security bonus, browser targets can be spawned in "offline mode" (using the
-  DevTools command [`Network.emulateNetworkConditions`](https://chromedevtools.github.io/devtools-protocol/tot/Network#method-emulateNetworkConditions).
-  Chrome targets with network conditions set to `offline` can't resolve any external URLs (e.g.
-  `https://`), neither entered as navigation URL nor contained within the HTML body.
+  If your template allows, you can disable JavaScript execution altogether (using the
+  DevTools command [`Emulation.setScriptExecutionDisabled`](https://chromedevtools.github.io/devtools-protocol/tot/Emulation/#method-setScriptExecutionDisabled))
+  with the `:disable_scripts` option:
 
       def chromic_pdf_opts do
-        [offline: true]
+        [disable_scripts: true]
       end
 
-  ### Disabling scripts
+  Note that this doesn't prevent other features like the `evaluate` option from working, it
+  solely applies to scripts being supplied by the rendered page itself.
+
+  ### Running in offline mode
 
-  Scripts are the biggest attack vector in a browser, potentially leading to many different
-  exploits in case of XSS, such as SSRF or even code execution in case of a v8 exploit. 
-  They can be disabled using the DevTools command [Emulation.setScriptExecutionDisabled](https://chromedevtools.github.io/devtools-protocol/tot/Emulation/#method-setScriptExecutionDisabled).
-  Note that this doesn't prevent other features, like the `evaluate` option from working,
-  it simply prevents scripts from being executed on the page.
+  To prevent your templates from accessing any remote hosts, the browser targets can be spawned
+  in "offline mode" (using the DevTools command [`Network.emulateNetworkConditions`](https://chromedevtools.github.io/devtools-protocol/tot/Network#method-emulateNetworkConditions)).
+  Chrome targets with network conditions set to `offline` can't resolve any external URLs (e.g.
+  `https://`), neither entered as navigation URL nor contained within the HTML body.
 
       def chromic_pdf_opts do
-        [disable_scripts: true]
+        [offline: true]
       end
 
   ### Chrome Sandbox in Docker containers
 
-  By default, ChromicPDF will allow Chrome to make use of its own ["sandbox" process jail](https://chromium.googlesource.com/chromium/src/+/master/docs/design/sandbox.md).
-  The sandbox tries to limit system resource access of the renderer processes to the minimum
-  resources they require to perform their task.
-
-  However, in Docker containers running Linux images (e.g. images based on Alpine), and which
-  are configured to run their main job as a non-root user, this causes Chrome to crash on startup
-  as it requires root privileges to enter the sandbox.
+  In Docker containers running Linux images (e.g. images based on Alpine), and which
+  are configured to run their main job as a non-root user, the sandbox may cause Chrome to crash
+  on startup as it requires root privileges.
```

Comment:
Maybe leave an advice here what do do then.

Reply:
It's below, doesn't show up because unchanged